<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="BUT_SEND">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TChat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>511</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddrChange">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>修改本机IP信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="MSG_TE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>接收者地址信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_IPADDR">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_PORT">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LB_IPAddr">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>72</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Port">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>112</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUT_SENDFILE">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
